### PR TITLE
 Add desc about watch-application's setup-appid

### PR DESCRIPTION
**Pull request by gotoextreme**
### Change Description ###

Add desc for watch-application setup-appid feature.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/435

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md [hunk 1]
```diff
@@ -431,8 +431,17 @@ For more information on the relationship between the elements, see the [element
 
   Watch application executable file path (string)
 
+- `setup-appid`
+
+  Watch application setup application ID (string)
+
+  If the value is 'watchface-editor', the system default editor is shown when the device enters the editing mode.
```
Comment (mijin85cho) [inline docs/application/tizen-studio/native-tools/manifest-text-editor.md]: Please change 'watchface-editor' to `watchface-editor`, using \`

File: docs/application/vstools/tools/manifest-editor.md [hunk 2]
```diff
@@ -360,8 +360,14 @@ For more information on the relationship between the elements, see the [element
 
     Tizen application type (available value: `dotnet`)
 
+-   `setup-appid`
+
+    Watch application setup application ID (string)
+
+    If the value is 'watchface-editor', the system default editor is shown when the device enters the editing mode.
```
Comment (mijin85cho) [inline docs/application/vstools/tools/manifest-editor.md]: Please change 'watchface-editor' to `watchface-editor`, using \`

File: docs/application/tizen-studio/native-tools/manifest-text-editor.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ There are 2 different ways to edit the `tizen-manifest.xml` file:
 
   ![Text Editor](./media/manifest_text.png)
 
-> **Note**  
+> **Note**
```
Comment (mijin85cho) [inline docs/application/tizen-studio/native-tools/manifest-text-editor.md]: Please change note style as below. Insert a blank line with > after **Note**.
```
> **Note**
>
> The ....
```

Comment (gotoextreme) [reply docs/application/tizen-studio/native-tools/manifest-text-editor.md]: Done

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/435

**Comment by mijin85cho** [comment]
Thanks for the update.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**